ding/policy/bc.py

```diff
@@ -195,3 +195,194 @@ def _get_train_sample(self, data: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
+            batch_size=32,
+            learning_rate=1e-4,
+        ),
+        collect=dict(unroll_len=1, ),
+        eval=dict(),
+        other=dict(replay_buffer=dict(replay_buffer_size=10000, )),
+    )
+
+    def _init_learn(self):
+        self._optimizer = SGD(
+            self._model.parameters(),
+            lr=1e-5,
+            momentum=0.9,
+        )
+        self._timer = EasyTimer(cuda=True)
+
+        #def lr_scheduler_fn(epoch):
+        #    if epoch <= self._cfg.learn.warmup_epoch:
+        #        return self._cfg.learn.warmup_lr / self._cfg.learn.learning_rate
+        #    else:
+        #        ratio = epoch // self._cfg.learn.decay_epoch
+        #        return math.pow(self._cfg.learn.decay_rate, ratio)
+        #self._lr_scheduler = LambdaLR(self._optimizer, lr_scheduler_fn)
+        #self._lr_scheduler.step()
+        self._learn_model = model_wrap(self._model, 'base')
+        self._learn_model.reset()
+        # self._mse_loss = nn.SmoothL1Loss()
+        self._mse_loss = nn.L1Loss()
+        # self._mse_loss = nn.MSELoss()
+
+    def _forward_learn(self, data):
+        if not isinstance(data, dict):
+            data = default_collate(data)
+        if self._cuda:
+            data = to_device(data, self._device)
+        self._learn_model.train()
+        with self._timer:
+            if self.cfg.eval.evaluator.cfg_type == 'MetricSerialEvaluatorDict':
+                obs, action = data
+            else:
+                obs, action = data['obs'], data['action'].squeeze()
+            # mu, sigma = self._learn_model.forward(obs, mode='compute_actor')['logit']
+            mu = self._eval_model.forward(data['obs'], mode='compute_actor')['action']
+            loss = self._mse_loss(mu, action)
+            # loss += self._mse_loss(sigma, torch.zeros_like(sigma))
+        forward_time = self._timer.value
+        with self._timer:
+            self._optimizer.zero_grad()
+            loss.backward()
+        backward_time = self._timer.value
+        with self._timer:
+            if self._cfg.learn.multi_gpu:
+                self.sync_gradients(self._learn_model)
+        sync_time = self._timer.value
+        self._optimizer.step()
+        cur_lr = [param_group['lr'] for param_group in self._optimizer.param_groups]
+        cur_lr = sum(cur_lr) / len(cur_lr)
+        return {
+            'cur_lr': cur_lr,
+            'total_loss': loss.item(),
+            'forward_time': forward_time,
+            'backward_time': backward_time,
+            'sync_time': sync_time,
+        }
+
+    def _monitor_vars_learn(self):
+        return ['cur_lr', 'total_loss', 'forward_time', 'backward_time', 'sync_time']
+
+    def _init_eval(self):
+        self._eval_model = model_wrap(self._model, wrapper_name='base')
+        self._eval_model.reset()
+
+    def _forward_eval(self, data):
+        if self.cfg.eval.evaluator.cfg_type == 'MetricSerialEvaluatorDict' or isinstance(data, torch.Tensor):
```

Comment:
What if you only have one evaluator_env in your config? Is there going to be a bug?